docs/source/installation-guide.rst

```diff
@@ -39,12 +39,59 @@ QUICK-21.03 CUDA version has been tested on the following GPU cards: A100, RTX20
 For the CUDA-MPI version we also recommend that only one CPU per GPU is used; this can be done by setting the number of processes (*e.g.*,
 in the *mpirun* command) equal to the number of CPUs.
 
-Installation
-------------
+Installation and Testing
+------------------------
 
-Using legacy build system
-^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Using the CMake build system
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CMake installation requires at least CMake/3.9.0 installed in the target machine. To install QUICK using CMake, one must first create a build  directory. Assuming you have created a directory named *builddir* in the ``QUICK_HOME`` directory and you want to install QUICK into directory ``QUICK_INSTALL``, use GNU compiler tool chain, and want to compile for Nvidia Volta microarchitecture, all QUICK versions can be configured and built as follows.
+
+.. code-block:: none
+
+	cd ${QUICK_HOME}/builddir
```

Comment:
Can we add info for users not familiar with CMake:

First you have to create a build directory. After installation you can safely delete this build directory if you want to save disk space.
cd ${QUICK_INSTALL}
mkdir build